### DISCUSSION TITLE
accesso app con wifi

### DISCUSSION THREAD

**Discussion by H2opolo** [indomus/forum#167] [HUB: Home Assistant]
Buonanotte
fino a qualche giorno fa accedevo all'app sia con la connessione tim che dal wifi casalingo utilizzando l'indirizzo di duckdns
avevo impostato la porta 443 all'esterno.
poi (sigh!) ho cambiato provider il quale mi ha installato un suo modem e mi ha aperto la porta esterna 8123.
da allora accedo all'app con la connessione dati del cellulare ma non con il wifi di casa
qualcuno sa aiutarmi?
ho provato a configurare anche l'indirizzo interno sull'app, ma nulla
devo dire che anche dal computer connesso in rete, se cerco di accedere tramite duckdns mi da errore

**Comment by snakuzzo** [comment]
Per quanto riguarda la configurazione interna, se hai HA configurato in https non potrai MAI accedere da APP con l'indirizzo LAN (es. https://192.168.1.2). Questo perché, come spiegato 100000000 di volte sia sul blog, che sulla chat telegram che sul forum, l'accesso in https con l'app "dipende" dai certificati. Se il certificato è generato su base FQDN ( es. pippo.duckdns.org ), quando proverai ad accedere puntando l'ip fallirà l'handshake. Da browser invece ti funzionerà e ti darà solo un warning che potrai bypassare con un click.
Se non riesci a puntare il tuo FQDN quando sei nella tua LAN probabilmente il modem non supporta il NAT loopback.

**Reply by H2opolo** [reply]
Grazie grazie per la risposta il problema è che io per 4 mesi ho utilizzato l'app accedendo utilizzando, sia con la rete dati del cellulare che con il wifi, l'indirizzo duck DNS con la porta 443. oggi se voglio fare la stessa cosa utilizzando la porta lo stesso indirizzo con la porta esterna 8123 non ci riesco. Mi dà login fallito e certificato SSL scaduto

**Reply by snakuzzo** [reply]
No però aspetta...questo è un altro discorso.
Se ti fallisce a causa del certificato scaduto, vuol dire che devi rinnovarlo.

**Reply by H2opolo** [reply]
Mi sono spiegato male.
Quando sono a casa se cerco di connettermi con l'app e la linea dati del cellulare è attiva (wifi spento) riesco ad accedere
Se, invece, sempre a casa cerco di connettermi con l'app ed il wifi acceso (linea dati cellulare spenta) mi da un messaggio che dice ssl scaduto. Ovviamente stessa istanza https:\\xxxx.duckdns.org:8123
Ho visto che da anche errore di login 
ho anche disinstallato e reinstallato duck dns ma niente
P.

**Reply by snakuzzo** [reply]
Controlla bene il messaggio d'errore.
Posta uno screenshot magari e soprattutto posta il log di HA nel momento in cui provi ad accedere

**Comment by fraygon** [comment]
ciao. da qualche giorno ho anch'io il tuo problema. Anche se è passato parecchio tempo ti chiedo come lo hai risolto perchè non io non ne vengo fuori.  
grazie

**Comment by H2opolo** [comment]
Purtroppo il mio provider non supporta  la tecnologia necessaria per poter
accedere con il WiFi da casa.
Ho fatto l'abbonamento a nabu casa 4.40 al mese ed ho risolto (sigh!)
P.

Il lun 26 lug 2021, 16:56 fraygon ***@***.***> ha scritto:

**Reply by fraygon** [reply]
ok. io ho fastweb e non mi ha mai creato problemi.  Non capisco cosa può essere successo. Comunque grazie.
